python/ray/tune/commands.py

```diff
@@ -130,35 +124,45 @@ def list_trials(experiment_path,
                 output=None,
                 filter_op=None,
                 info_keys=None,
-                result_keys=None):
+                limit=None,
+                desc=False):
     """Lists trials in the directory subtree starting at the given path.
 
     Args:
         experiment_path (str): Directory where trials are located.
             Corresponds to Experiment.local_dir/Experiment.name.
-        sort (str): Key to sort by.
+        sort (list): Keys to sort by.
         output (str): Name of file where output is saved.
         filter_op (str): Filter operation in the format
             "<column> <operator> <value>".
         info_keys (list): Keys that are displayed.
-        result_keys (list): Keys of last result that are displayed.
+        limit (int): Number of rows to display.
+        desc (bool): Sort ascending vs. descending.
     """
     _check_tabulate()
     experiment_state = _get_experiment_state(
         experiment_path, exit_on_fail=True)
 
-    checkpoint_dicts = experiment_state["checkpoints"]
-    checkpoint_dicts = [flatten_dict(g) for g in checkpoint_dicts]
+    checkpoints = experiment_state["checkpoints"]
+
+    # import ipdb; ipdb.set_trace()
+
+    checkpoint_dicts = []
+    for g in checkpoints:
+        config = g.pop("config")
+        config = flatten_dict(config)
+        last_result = g.pop("last_result")
+        last_result = flatten_dict(last_result)
+        g = flatten_dict(g)
+        combined_dict = {**config, **last_result, **g}
+        checkpoint_dicts.append(combined_dict)
```

Comment:
what about something like

```python
UNNEST_KEYS = ["config", "last_result"] # global

...
def list_trials(...):
    ...
    checkpoint_dicts = []
    for g in checkpoints:
        for key in UNNEST_KEYS:
            if key not in g:
                continue
            unnest_dict = flatten_dict(g.pop(key))
            g.update(key)
        checkpoint_dicts.append(g)
```

Reply:
are `config` and `last_result` the only nested dicts in `g`? or should we do `g = flatten_dict(g)` right before `checkpoint_dicts.append(g)`